src/navigate-pop/tiling-stacking-windows.md

```diff
@@ -0,0 +1,86 @@
+# Manipulating Windows
+
+This section describes the Auto Tiling, Stacking, and window adjustment features in Pop!\_OS.
+
+---
+## Window Adjustment Mode
+
+Adjustment mode allows users to easily manipulate the placement of windows using keyboard shortcuts.
+
+Place a window into Management mode by positioning your cursor over the window, and then pressing `SUPER` + `Enter`. The window will be highlighted in yellow. You can now manipulate the window with `🠐` `🠑` `🠒` `🠓` keys in combination with other keys.
+
+![Adjustment Mode Manipulation](/images/tiling-stacking-windows/adjustment-mode-manipulation.gif)
+
+All keyboard shortcuts available in adjustment mode can be viewed and modified in Settings ➞ Keyboard ➞ View and Customize Shortcuts ➞ Move, resize, and swap windows.
```

Comment:
The default shortcuts can be viewed a little easier in the top-right drop-down, I wonder if that would be worth mentioning?

Reply:
That was not what I was referring to. Those per-app shortcuts are unrelated to tiling and probably off-topic for this page; I''m un-resolving this conversation.

I was talking about the Pop Shell menu in the top-right of the screen. A user can click the View All menu entry (or just the shortcuts entry above that) to open a window that shows all of the tiling/stacking-related window management shortcuts.

Reply:
Fixed.